src/pybind/mgr/dashboard/CMakeLists.txt

```diff
@@ -4,12 +4,20 @@ install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
   PATTERN "frontend/*" EXCLUDE
   PATTERN ".*" EXCLUDE)
 
+include(ExternalProject)
+ExternalProject_Add(grafonnet-lib
+  DOWNLOAD_DIR ${PROJECT_SOURCE_DIR}
```

Comment:
please do not create files in `${PROJECT_SOURCE_DIR}`. also this option only applies to "URL Download". see https://cmake.org/cmake/help/latest/module/ExternalProject.html

but you are using Git here.